recipes/stateless-service/06-task.yml

```diff
@@ -0,0 +1,111 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: build-project
+spec:
+  workspaces:
+  - name: maven-repo
+  inputs:
+    params:
+    - name: directory
+      default: .
+    - name: cache
+      default: empty-dir-volume
+    resources:
+    - name: src
+      type: git
+  steps:
+   - name: set-source-permissions
+     image: bash
+     command:
+     - chmod
+     - -R
+     - a+w
+     - .
+     workingDir: /workspace/src/$(inputs.params.directory)
+   - name: build-package
+     image: quay.io/quarkus/centos-quarkus-maven:20.1.0-java11
+     command:
+     - mvn
+     - clean
+     - package
+     - -Dquarkus.package.uber-jar
+     - -Dmaven.repo.local=$(workspaces.maven-repo.path)
+     workingDir: /workspace/src/$(inputs.params.directory)
+   - name: build-image
+     image: quay.io/openshift-pipeline/s2i
+     command:
+     - s2i 
+     - build 
+     - target/
+     - quay.io/kiegroup/kogito-quarkus-jvm-ubi8:latest 
+     - process-quarkus-example
+     workingDir: /workspace/src/$(inputs.params.directory)
+     volumeMounts:
+     - name: $(inputs.params.cache)
+       mountPath: /builder/home/.m2
+       subPath: m2-cache
+     - name: dind-socket
+       mountPath: /var/run/
+   - name: tag-image
+     image: docker:18.05-dind
+     command:
+     - docker 
+     - tag
+     - process-quarkus-example
+     - quay.io/mmagnani/process-quarkus-example
```

Comment:
I'll give you permissions to the `m88i` repo, or we can use Kogito examples of images that we build on `kiegroup.`